server/middlewares/stats.js

```diff
@@ -12,6 +12,12 @@ function getSlicedDate() {
     .split('-')
 }
 
+const isSameOrigin = (urlString1, urlString2) => {
+  const url1 = new URL(urlString1)
+  const url2 = new URL(urlString2)
+  return url1.origin === url2.origin
```

Comment:
I think the `host` attribute is safer since it strips the `https://` part which isn't really a problem now, since we do https redirect, but I would rather ignore it in the comparison to stay on the safe side.

Reply:
although well, calling a function `isSameOrigin` and comparing `host`s seems a bit fishy to me, I leave it up to your taste, perhaps adding a comment explaining the rationale behind using `host` instead of `origin` would do it

Reply:
I would rather not use external library for this so I chose the first option.

Reply:
but this still has the problem that if somebody accesses adalite.io through www.adalite.io and makes a transaction, the backend would wrongly recognize it as a foreign transaction

Reply:
but a regex would do it as well, of course: https://stackoverflow.com/questions/3738282/javascript-strip-words-from-url

Reply:
Thanks for the example, I see now why you wanted to normalize the URL's. Probably the easiest solution is to use the library.